include/xtensor-fftw/xtensor-fftw_config.hpp

```diff
@@ -10,8 +10,8 @@
 #define XTENSOR_FFTW_CONFIG_HPP
 
 #define XTENSOR_FFTW_VERSION_MAJOR 0
-#define XTENSOR_FFTW_VERSION_MINOR 2
-#define XTENSOR_FFTW_VERSION_PATCH 6
+#define XTENSOR_FFTW_VERSION_MINOR 3
+#define XTENSOR_FFTW_VERSION_PATCH 0
```

Comment:
This should be updated when releasing only.

Reply:
Done.